modules/candidate_profile/jsx/CandidateInfo.js

```diff
@@ -0,0 +1,183 @@
+     *
+     * @param {array} visits - List of visits in the format returned by the LORIS API.
+     *
+     * @return {object} - A React element containing a comma separated list of links.
+     */
+    getVisitList(visits) {
+        let visitlinks = visits.map( (visit) => {
+            const sessionID = this.props.VisitMap[visit.Meta.Visit];
+            const candID = this.props.Candidate.Meta.CandID;
+            return <a
+                href={
+                    this.props.BaseURL
+                    + '/instrument_list/?candID=' + candID
+                    + '&sessionID=' + sessionID}
+                key={visit.Meta.Visit}
+                >{visit.Meta.Visit}</a>;
+        });
+        return <div>
+            {
+                // Equivalent of .join(', ') that doesn't convert the React
+                // element into the string [object Object].
+                // See https://stackoverflow.com/questions/33577448/is-there-a-way-to-do-array-join-in-react
+                visitlinks.reduce(
+                    (acc, el) => {
+                        if (acc === null) {
+                            return [el];
+                        }
+                        return [acc, ', ', el];
+                    },
+                    null,
+                )
+            }
+            </div>;
+    }
+
+    /**
+     * Render the React component
+     *
+     * @return {object} - The rendered react component
+     */
+    render() {
+        const subprojects = this.getSubprojects(this.props.Visits);
+        const subprojlabel = subprojects.length == 1 ? 'Subproject'
+            : 'Subprojects';
+
+        const data = [
+            {
+                label: 'PSCID',
+                value: this.props.Candidate.Meta.PSCID,
+            },
+            {
+                label: 'DCCID',
+                value: this.props.Candidate.Meta.CandID,
+            },
+            {
+                label: 'Date of Birth',
+                value: this.props.Candidate.Meta.DoB,
+            },
+            {
+                label: 'Age',
+                value: this.calcAge(this.props.Candidate.Meta.DoB),
+            },
+            {
+                label: 'Sex',
+                value: this.props.Candidate.Meta.Sex,
+            },
+            {
+                label: 'Project',
+                value: this.props.Candidate.Meta.Project,
+            },
+            {
+                label: subprojlabel,
+                value: subprojects,
+            },
+            {
+                label: 'Site',
+                value: this.props.Candidate.Meta.Site,
+            },
+            {
+                label: 'Visits',
+                value: this.getVisitList(this.props.Visits),
+                width: '12em',
+            },
+        ];
+
+        const cardInfo = data.map((info, index) => {
+            const cardStyle = {
+                width: info.width || '6em',
+                padding: '1em',
+                margin: 0,
```

Comment:
could we add a left and right margin? info.value is really close to each other if the string is long 
<img width="196" alt="Screenshot 2020-02-26 15 03 36" src="https://user-images.githubusercontent.com/17415878/75383040-9e7e0e00-58a9-11ea-8a93-2dba6703b995.png">

Reply:
I'm not sure it'll help much if the string is even longer, but done..